creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -671,132 +676,137 @@ class SnapbackSM {
   }
 
   /**
-   * Given map(replica set node => userWallets[]), retrieves clock values for every (node, userWallet) pair
+   * Given map(replica set node => userWallets[]), retrieves user info for every (node, userWallet) pair
    * @param {Object} replicaSetNodesToUserWalletsMap map of <replica set node : wallets>
    * @param {Set<string>} unhealthyPeers set of unhealthy peer endpoints
    * @param {number?} [maxUserClockFetchAttempts=10] max number of attempts to fetch clock values
    *
-   * @returns {Object} map of peer endpoints to (map of user wallet strings to clock value of replica set node for user)
+   * @returns {Object} map(replica => map(wallet => { clock, filesHash }))
    */
-  async retrieveClockStatusesForUsersAcrossReplicaSet(
-    replicaSetNodesToUserWalletsMap,
+  async retrieveUserInfoFromReplicaSet(
+    replicasToWalletsMap,
     unhealthyPeers,
     maxUserClockFetchAttempts = 10
   ) {
-    const replicaSetNodesToUserClockValuesMap = {}
-
-    const replicaSetNodes = Object.keys(replicaSetNodesToUserWalletsMap)
+    const replicasToUserInfoMap = {}
 
     // TODO change to batched parallel requests
+    const replicas = Object.keys(replicasToWalletsMap)
     await Promise.all(
-      replicaSetNodes.map(async (replicaSetNode) => {
-        replicaSetNodesToUserClockValuesMap[replicaSetNode] = {}
+      replicas.map(async (replica) => {
+        replicasToUserInfoMap[replica] = {}
 
-        const replicaSetNodeUserWallets =
-          replicaSetNodesToUserWalletsMap[replicaSetNode]
-        const { timestamp, signature } = generateTimestampAndSignature(
-          { spID: this.spID },
-          this.delegatePrivateKey
-        )
+        const walletsOnReplica = replicasToWalletsMap[replica]
 
         const axiosReqParams = {
```

Comment:
this should be let

Reply:
nah, we aren't reassigning this variable, just modifying properties on the object. variable pointer is unchanged

Reply:
tbh we should probably be using let if we're modifying any variables, since consts imply that nothing about the variable would be changed. we can get away with the linter passing for key-value setting in objects cc @dmanjunath

Reply:
nah, [from MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const):
> The value of a constant can't be changed through reassignment (i.e. by using the assignment operator), and it can't be redeclared (i.e. through a variable declaration). **However, if a constant is an object or array its properties or items can be updated or removed.**

only reason to use `let` here would be to enable re-assignment which i am not doing. i think `const` is correct

Reply:
hmmm yeah ok. the reference obj is still the same, just with updated properties. sg 👍